vocabularies/UI.xml

```diff
@@ -1832,6 +1832,56 @@ This is in contrast to [DataFieldForIntentBasedNavigation](#DataFieldForIntentBa
         </Property>
       </ComplexType>
 
+      <Term Name="Recommendations" Type="Edm.ComplexType" Nullable="false">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
```

Comment:
Do CAP or RAP plan to use this as an instance annotation?

If not, I'd remove this tag now and add it once it is needed.